src/traces/scattermapbox/defaults.js

```diff
@@ -104,12 +67,14 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     var clusterEnabled = coerce('cluster.enabled', clusterEnabledDflt);
 
     if(clusterEnabled || subTypes.hasText(traceOut)) {
+        var layoutFontFamily = layout.font.family;
+
         handleTextDefaults(traceIn, traceOut, layout, coerce,
             {
                 noSelect: true,
                 noFontVariant: true,
                 font: {
-                    family: supportedFonts.indexOf(layout.font.family) !== -1 ? layout.font.family : 'Open Sans Regular',
+                    family: isSupportedFont(layoutFontFamily) ? layoutFontFamily : 'Open Sans Regular',
```

Comment:
ok, naive question because I don't quite understand -- if the user passes just `Metropolis` (for example) as the font family, won't that register as "not supported" because the string `Metropolis` is not in the supported fonts list, and so it will get replaced with `Open Sans`?

Reply:
Now that we have `weight` and `style` options, would it make sense to change that behavior? Since a user could pass `family: 'Metropolis', weight: 'bold'` and we could use that to determine they want the font face `Metropolis Bold`.

Reply:
Actually I was thinking about that and I'd be happy to work on it in a separate PR.
Obviously we don't want to break previous mocks. 
Instead we could work on "adding support for minimal family names in `mapbox`".

Reply:
ok, that's fine to do in another PR. Would be nice to have!

Reply:
Tracked in #6993.